docs/en-US/component/upload.md

```diff
@@ -89,53 +89,106 @@ upload/manual
+| name                          | key name for uploaded file.                                                                                                                                                           | ^[string]                                                                                                | file                                                                                                               |
+| with-credentials              | whether cookies are sent.                                                                                                                                                             | ^[boolean]                                                                                               | false                                                                                                              |
+| show-file-list                | whether to show the uploaded file list.                                                                                                                                               | ^[boolean]                                                                                               | true                                                                                                               |
+| drag                          | whether to activate drag and drop mode.                                                                                                                                               | ^[boolean]                                                                                               | false                                                                                                              |
+| accept                        | accepted [file types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-accept), will not work when `thumbnail-mode === true`.                                     | ^[string]                                                                                                | ''                                                                                                                 |
+| on-preview                    | hook function when clicking the uploaded files.                                                                                                                                       | ^[Function]`(uploadFile: UploadFile) => void`                                                            | () => void                                                                                                         |
```

Comment:
`() => void` -> `—`

What do you think?

Reply:
agree